notebooks/exploring_explorer.livemd

```diff
@@ -594,48 +623,58 @@ DataFrame.ungroup(grouped)
 But what we care about the most is aggregating! Let's see which country has the max `per_capita` value.
 
 ```elixir
-grouped |> DataFrame.summarise(per_capita: [:max]) |> DataFrame.arrange(desc: :per_capita_max)
+grouped
+|> DataFrame.summarise(max_per_capita: max(per_capita))
+|> DataFrame.arrange(desc: max_per_capita)
 ```
 
-Qatar it is. You can use the following aggregations:
+Qatar it is.
 
-* `:min` - Take the minimum value within the group. See `Explorer.Series.min/1`.
-* `:max` - Take the maximum value within the group. See `Explorer.Series.max/1`.
-* `:sum` - Take the sum of the series within the group. See `Explorer.Series.sum/1`.
-* `:mean` - Take the mean of the series within the group. See `Explorer.Series.mean/1`.
-* `:median` - Take the median of the series within the group. See `Explorer.Series.median/1`.
-* `:first` - Take the first value within the group. See `Explorer.Series.first/1`.
-* `:last` - Take the last value within the group. See `Explorer.Series.last/1`.
-* `:count` - Count the number of rows per group.
-* `:n_unique` - Count the number of unique rows per group.
+You may noticed that we are using `max/1` inside the `summarise` macro. This is possible because we expose all functions from the `Series` module. You can use the following aggregations inside summarise:
+
+* `min/1` - Take the minimum value within the group. See `Explorer.Series.min/1`.
+* `max/1` - Take the maximum value within the group. See `Explorer.Series.max/1`.
+* `sum/1` - Take the sum of the series within the group. See `Explorer.Series.sum/1`.
+* `mean/1` - Take the mean of the series within the group. See `Explorer.Series.mean/1`.
+* `median/1` - Take the median of the series within the group. See `Explorer.Series.median/1`.
+* `first/1` - Take the first value within the group. See `Explorer.Series.first/1`.
+* `last/1` - Take the last value within the group. See `Explorer.Series.last/1`.
+* `count/1` - Count the number of rows per group.
+* `n_unique/1` - Count the number of unique rows per group.
 
 The API is similar to `mutate`: you can use keyword args or a map and specify aggregations to use.
 
 ```elixir
-grouped |> DataFrame.summarise(per_capita: [:max, :min], total: [:min])
+DataFrame.summarise(grouped, min_per_capita: min(per_capita), min_total: min(total))
 ```
 
 Speaking of `mutate`, it's 'group-aware'. As are `arrange`, `distinct`, and `n_rows`.
 
 ```elixir
-DataFrame.arrange(grouped, desc: :total)
+DataFrame.mutate(grouped, total_window_sum: window_sum(total, 3), rows_in_group: count(country))
 ```
 
-In case you need more complex operations, like calculations inside groups, you can use the
-more flexible `Explorer.DataFrame.summarise_with/2` function.
+In case you need more complex operations, like calculations inside groups, you can use multiple lines, like this:
 
 ```elixir
-DataFrame.summarise_with(grouped, fn df ->
-  max = Series.max(df["per_capita"])
+DataFrame.summarise(
+  grouped,
+  (
+    ^max = max(per_capita)
```

Comment:
Hrm, this is very confusing :( We should probably not allow `=` inside queries for now.

I would rewrite this to:

summarize(per_capita_max: max(per_capita), greater_than_9: max(per_capita) > 9.0)